ext/compressors/iaa/iaacodec/include/IAACompressionCodecDeflate.h

```diff
@@ -0,0 +1,216 @@
+#pragma once
+
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+#include <unistd.h>
+#include <chrono>
+#include <sys/mman.h>
+#include <iostream>
+#include <atomic>
+#include <qpl/qpl.h>
+#include <vector>
+#include <map>
+#include <x86intrin.h>
+#include <memory>
+#include <thread>
+#include <wiredtiger.h>
+#include <wiredtiger_ext.h>
+namespace DB
```

Comment:
The namespace `DB` is very generic. Should we think of a different name related to `iaa`?

Reply:
OK, We will change DB to DB::IAA